tests/test_edgeql_ddl.py

```diff
@@ -5065,6 +5065,51 @@ async def test_edgeql_ddl_module_02(self):
             CREATE TYPE spam::Test;
         ''')
 
+    async def test_edgeql_ddl_module_03(self):
+        await self.assert_query_result(
+            r'''
+            select _test::abs(-1)
+            ''',
+            [1]
+        )
+        await self.con.execute('''\
+            CREATE MODULE _test
+        ''')
+        with self.assertRaisesRegex(
+                edgedb.InvalidReferenceError,
+                "'_test::abs' does not exist"):
+            await self.con.execute('''\
+            select _test::abs(-1)
+            ''')
+
+    async def test_edgeql_ddl_module_04(self):
+        async with self.assertRaisesRegexTx(
+                edgedb.UnknownModuleError,
+                "module 'foo' is not in this schema"):
+            await self.con.execute('''\
+                CREATE MODULE foo::bar;
+            ''')
+
+        await self.con.execute('''\
+            CREATE MODULE foo;
+            CREATE MODULE foo::bar;
+            CREATE TYPE foo::bar::Baz;
+        ''')
+
+        await self.assert_query_result(
+            r'''
+            select foo::bar::Baz
+            ''',
+            []
+        )
+        await self.assert_query_result(
+            r'''
+            with module foo::bar
```

Comment:
Can we have some more tests for setting aliases at various depths (both via `with` and `set module`) and also some shadowing tests.

Reply:
Added more tests for setting aliases, fixed a couple bugs.

There's only really one shadowing behavior (shadowing a module under `std`) and it had a test in the first pass